### PR TITLE
Media3 guide

### PR DESCRIPTION
**Pull request by hovig-theo**

### CODE REVIEW

File: theoplayer/how-to-guides/android/media3/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Getting started with Media3 on Android
+
+<Intro />
```
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/media3/getting-started.mdx:3]: This component doesn't exist. (I assume you copied this from #143, where it's an imported component to have the same introduction across Web and Android.)

Remove it, and write your own introduction inline. 😉

File: theoplayer/how-to-guides/android/media3/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    .Builder(typedSource)
+    .build();
+
+    theoPlayerView.getPlayer().setSource(sourceDescription);
+    ```
+  </TabItem>
+  <!-- prettier-ignore-end -->
+</Tabs>
+
+By default, the Media3PlayerIntegration will play all type of sources except HESP & MILLICAST.
```
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/media3/getting-started.mdx:84]:
```suggestion
By default, the `Media3PlayerIntegration` will play all type of sources except HESP and Millicast.
```

File: theoplayer/how-to-guides/android/media3/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      implementation("com.theoplayer.theoplayer-sdk-android:core:8.+")
+      // highlight-next-line
+      implementation("com.theoplayer.theoplayer-sdk-android:integration-media3:8.+")
+    }
+    ```
+  </TabItem>
+  <!-- prettier-ignore-end -->
+</Tabs>
+
+### Add the Media3 Integration to the player.
```
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/media3/getting-started.mdx:39]:
```suggestion
### Add the Media3 Integration to the player
```

File: theoplayer/how-to-guides/android/media3/getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+<Intro />
+
+## Usage
+
+1. Follow [our Getting Started guide](../../../getting-started/01-sdks/02-android/00-getting-started.mdx)
+   to set up THEOplayer in your Android app.
+2. Add the Media3 Integration as a dependency in your module-level `build.gradle` file:
+3. Add the Media3 Integration to the player.
+
+### Add the Media3 Integration dependency
```
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/media3/getting-started.mdx:12]: Nit-pick: write "integration" in lowercase.
```suggestion
### Add the Media3 integration dependency
```
(This occurs a couple of times in this file.)

**Review by MattiasBuelens — CHANGES_REQUESTED** [review]

File: theoplayer/how-to-guides/android/media3/getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    .Builder(typedSource)
+    .build();
+
+    theoPlayerView.getPlayer().setSource(sourceDescription);
+    ```
+  </TabItem>
+  <!-- prettier-ignore-end -->
+</Tabs>
+
+By default, the `Media3PlayerIntegration` will play all types of sources except HESP and MILLICAST.
```
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/media3/getting-started.mdx:89]: Almost! 😛 
```suggestion
By default, the `Media3PlayerIntegration` will play all types of sources except HESP and Millicast.
```

**Review by MattiasBuelens — APPROVED** [review]
LGTM.

Let's wait for the Millicast PR to land first, so the whole platform-specific how-to guide thingy works correctly. 😉

**Comment by GillesMoris-THEO** [comment]
Wait to merge, reviewing

File: theoplayer/how-to-guides/android/media3/getting-started.mdx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+})
+theoplayerView.player.addIntegration(media3PlayerIntegration);
+```
+
+## Known limitations
+
+As this integration is still under development, there are currently some known limitations and features that are still under development and not yet supported:
+- The preload, ABR, network, metrics latency and caching / offline playback APIs are not yet implemented and return dummy values.
+- HESP/THEOlive and Millicast playback are not yet supported, and will fall back to our existing playback pipeline.
+- TextTrack.cues, TextTrack.activeCues and cue-related events are not yet supported. Although subtitles do render correctly to the screen, the content of these subtitles is not yet exposed through our API.
```
Comment (MattiasBuelens) [inline theoplayer/how-to-guides/android/media3/getting-started.mdx:108]:
```suggestion
- `TextTrack.cues`, `TextTrack.activeCues` and cue-related events are not yet supported. Although subtitles do render correctly to the screen, the content of these subtitles is not yet exposed through our API.
```

**Review by MattiasBuelens — APPROVED** [review]

**Review by GillesMoris-THEO — APPROVED** [review]

**Pull request merged**